### DISCUSSION TITLE
Android builds?

### DISCUSSION THREAD

**Discussion by Gugui322** [cataclysmbn/Cataclysm-BN#2157] [Q&A] [answered]
Since this [commit](https://github.com/cataclysmbnteam/Cataclysm-BN/commit/3deef06a4286a822e166872e7bcac004470dd13c) there is no android builds.

**Comment by scarf005** [comment, marked as answer]
now fixed from [2022-11-04](https://github.com/cataclysmbnteam/Cataclysm-BN/releases/tag/cbn-experimental-2022-11-04-1021) via #2160
